packages/create-sitecore-jss/src/templates/angular/package.json

```diff
@@ -34,7 +34,9 @@
     "build:server": "cross-env-shell ng run <%- appName %>:server:production --output-path=$npm_package_config_buildArtifactsPath/server --output-hashing=none --bundle-dependencies true",
     "postbuild:server": "move-cli ./dist/server/main.js ./dist/server.bundle.js && del-cli ./dist/server",
     "bootstrap": "ts-node --project src/tsconfig.webpack-server.json scripts/bootstrap.ts",
-    "graphql:update": "ts-node --project src/tsconfig.webpack-server.json ./scripts/update-graphql-fragment-data.ts"
+    "graphql:update": "ts-node --project src/tsconfig.webpack-server.json ./scripts/update-graphql-fragment-data.ts",
+    "install-pre-push-hook": "node ./scripts/install-pre-push-hook.js",
+    "prepush": "yarn lint --fix"
```

Comment:
Do we need this additional 'prepush' command with the --fix or can we just call the existing 'lint' command? If we do keep this, would need to use npm.

Reply:
hmm yeah right we don't need it. WE can directly use the lint script.